.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - 6.10.2
+before_script:
+  - npm install -g gulp
+install: true
```

Comment:
I meant remove install: true, that setting means travis should not install node_modules
I suggest reading this - https://docs.travis-ci.com/user/customizing-the-build